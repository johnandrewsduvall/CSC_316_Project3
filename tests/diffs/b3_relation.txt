1d0
< $
15d13
< $
