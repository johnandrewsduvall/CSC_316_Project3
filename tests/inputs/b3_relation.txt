$
A
E
G
$
F
C
D
E
B
$
I
H
$
$
