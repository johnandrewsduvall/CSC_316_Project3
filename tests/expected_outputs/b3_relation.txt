relation A G
relation F B
relation I H
relation F I
